technical/trendline.py

```diff
@@ -5,127 +5,86 @@
 """
 
 
-def gentrends(dataframe, field="close", window=1 / 3.0, charts=False):
-    """
-    Returns a Pandas dataframe with support and resistance lines.
-
-    :param dataframe: incomming data matrix
-    :param field: for which column would you like to generate the trendline
-    :param window: How long the trendlines should be. If window < 1, then it
-                   will be taken as a percentage of the size of the data
-    :param charts: Boolean value saying whether to print chart to screen
-    """
-
-    x = dataframe[field]
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
Technical doesn't depend on matplotlib - and we don't intend to.

The way it was previously made this optional - where the code would fail if matplotlib isn't installed.
Now it's failing all the time unless we depend on matplotlib.

Please keep the matplotlib import where it was.

Reply:
Sure, I will make the desired changes and then redirect to you.